helm/cloud-provider-azure/templates/cloud-provider-azure.yaml

```diff
@@ -48,11 +48,11 @@ rules:
     verbs:
       - "*"
   - apiGroups:
-      - ""
+      - "crd.projectcalico.org"
     resources:
-      - nodes/status
+      - blockaffinities
     verbs:
-      - patch
+      - "*"
```

Comment:
We should add our own rule, not modifying the existing one if possible:
```yaml
- apiGroups:
  - "crd.projectcalico.org"
  resources:
  - blockaffinities
  verbs:
  - "*"
```

Reply:
This was done by mistake, fixed.